File: Leetcode-Problems/Find N Unique Integers Sum up to Zero.cpp
class Solution
{
public:
    vector<int> sumZero(int n)
    {
        vector<int> arr;
        int temp = n / 2;
        while (temp > 0)
        {
            arr.push_back(-temp);
            arr.push_back(temp);
            temp--;
        }
        if (n % 2 != 0)
        {
            arr.push_back(0);
        }
        sort(arr.begin(), arr.end());
        return arr;
    }
};

File: Leetcode-Problems/Find N Unique Integers Sum up to Zero.java
class Solution {
    public int[] sumZero(int n) {
        int[] arr = new int[n];
        int i = 0;
        int temp = n / 2;
        while (temp > 0) {
            arr[i++] = -temp;
            arr[i++] = temp;
            temp--;
        }
        if (n % 2 != 0) {
            arr[i] = 0;
        }
        return arr;
    }
}